test/e2e/main/main_test.go

```diff
@@ -83,7 +84,9 @@ func TestUpdateOperatorPassword(t *testing.T) {
 	spec.Name = name
 	// Register it
 	testKube.CreateInfinispan(spec, tutils.Namespace)
-	defer testKube.DeleteInfinispan(spec, tutils.SinglePodTimeout)
+	spec.Labels = map[string]string{"test-name": t.Name()}
```

Comment:
Also labels assigned after Infinispan CR creation.
@durgeshanaokar please validate all possible places for this issue.